trace/opencensus/trace/ext/django/config.py

```diff
@@ -31,6 +31,7 @@
     'ZIPKIN_EXPORTER_SERVICE_NAME': 'my_service',
     'ZIPKIN_EXPORTER_HOST_NAME': 'localhost',
     'ZIPKIN_EXPORTER_PORT': 9411,
+    'BLACKLIST_PATHS': ['/_ah/health'],
```

Comment:
These should probably be in alphabetical order.  Also, can you add a comment next to the url with a link to https://cloud.google.com/appengine/docs/flexible/python/how-instances-are-managed#health_checking .

Reply:
Also, we should document that these are literal string matches, rather than regular expressions or something like that.

Reply:
Done.